### DISCUSSION TITLE
[FEAT] Give confirmation of patched packages

### DISCUSSION THREAD

**Discussion by ivnnv** [pnpm/pnpm#15991] [Ideas]
### Contribution

- [X] I'd be willing to implement this feature ([contributing guide](https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md))

### Describe the user story

Hi, it seems theres indeed a message logged when any patches havent been applied:
`The following patches were not applied`

### Describe the solution you'd like

It would be nice to also log when patches have been applied a-la `patch-package` as it gives you peace of mind when you read it (specially on CI/CD environments).

### Describe the drawbacks of your solution

I dont think theres any drawbacks on having just a new log line on install

### Describe alternatives you've considered

this is pretty straightforward, the alternative is just switch back to `patch-package` lol

**Comment by zkochan** [comment]
What does patch-package print out?

**Comment by ivnnv** [comment]
hey @zkochan, thanks for the early response, this is the output:

```
> patch-package

patch-package 8.0.0
Applying patches...
bull-arena@4.1.0 ✔
```

**Comment by ivnnv** [comment]
@zkochan friendly ping, I could be able to implement this feature if its just loggin [here](https://github.com/pnpm/pnpm/blob/19c4b4f9737491acaf1a80ac8a1e65f66e7d5351/pkg-manager/resolve-dependencies/src/index.ts#L330)

**Comment by zkochan** [comment]
ok, go ahead

**Comment by altaywtf** [comment]
I'm working on a fix for this.

Written by an agent (Claude Code, Opus 5.5).

**Comment by altaywtf** [comment]
A fix is ready for review in https://github.com/pnpm/pnpm/pull/15846. CI is green.

Written by an agent (Claude Code, Opus 5.5).
